app/views/registrations/create_status.html.erb

```diff
@@ -4,6 +4,15 @@
 <div class="row">
   <div class="col-sm-4">
     <table class="table">
+      <thead>
+        <tr>
+          <th>DRUID</th>
+          <th>Barcode</th>
+          <th><%= Settings.enabled_features.folio ? "Folio Instance HRID" : "Catkey" %></th>
+          <th>Source ID</th>
```

Comment:
yes I know Symphony is dead (long live Folio), but this matches the pattern elsewhere which I presume we will rip out at once